libraries/cmake/source/sleuthkit/config/windows/tsk/tsk_config.h

```diff
@@ -0,0 +1,285 @@
+/* tsk/tsk_config.h.  Generated from tsk_config.h.in by configure.  */
```

Comment:
How was this file generated?

Reply:
took it from a "linux/tsk" with one change - disabled option HAVE_UNISTD_H
WIth next commit, I did review each option.